src/Sylius/Bundle/ResourceBundle/spec/Form/Type/IdentifierToResourceTypeSpec.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\ResourceBundle\Form\Type;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Bundle\ResourceBundle\Form\DataTransformer\IdentifierToResourceTransformer;
+use Sylius\Component\Resource\Metadata\MetadataInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\Form\Test\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class IdentifierToResourceTypeSpec extends ObjectBehavior
+{
+    function let(RepositoryInterface $repository, MetadataInterface $metadata)
+    {
+        $this->beConstructedWith($repository, $metadata);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\ResourceBundle\Form\Type\IdentifierToResourceType');
+    }
+
+    function it_build_a_form(FormBuilderInterface $builder)
+    {
+        $builder->addModelTransformer(
+            Argument::type(IdentifierToResourceTransformer::class)
+        )->shouldBeCalled();
+
+        $this->buildForm($builder, array(
+            'identifier' => 'identifier',
+        ));
+    }
+
+    function it_has_options(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults(array(
+            'identifier' => 'id'
+        ))->willReturn($resolver);
+
+        $resolver->setAllowedTypes('identifier', 'string')->willReturn($resolver);
+
+        $this->configureOptions($resolver);
+    }
+
+    function it_has_a_parent()
+    {
+        $this->getParent()->shouldReturn('entity');
+    }
+
+    function it_has_a_name($metadata)
+    {
+        $metadata->getName()->willReturn('name');
+        $metadata->getApplicationName()->willReturn('sylius');
+        $this->getName()->shouldReturn('sylius_name_from_identifier');
```

Comment:
It would be nice to have background seperated from assertions by some blank line.